lib/http/request.js

```diff
@@ -190,7 +190,7 @@ const create = async function (method, path, options, hooks = {}) {
   })
 
   if (options.queryParams) {
-    path += `?${QueryString.stringify(options.queryParams)}`
+    path += `${path.includes('?') ? '&' : '?'}${QueryString.stringify(options.queryParams)}`
```

Comment:
I think we also need to account for if `path` already ends in a `&`. We wouldn't want to have `?foo=bar&&fizz=buzz`.

Some options to address this:
* strip any trailing `&` before processing anything
* don't add a `&` if one already exists
* parse the path's query string into an object, merge `options.queryParams` into the object, then stringify and re-attach to the path

Reply:
i took the third approach